source/docs/templates.md

```diff
@@ -101,4 +101,10 @@ Though it may be easier to use partials:
 <%- partial('header', {}, {cache: true});
 ```
 
-Don't use fragment caching when the `relative_link` setting has been enabled. This may cause issues because relative links can and probably will be different depending on the pages they appear in.
+{% note warn %}
+`fragment_cache()` will cache the rendered result and output the cached result to other pages.
+This should only be used on partials that are expected **not** to change across different pages.
+Otherwise, it should not be enabled.
```

Comment:
what I meant previously is ignore the new lines, so L105-107 can be in a single line, same goes to L108-109.

Reply:
Rebased & updated.